MANIFEST

```diff
@@ -834,7 +834,7 @@ cpan/ExtUtils-MakeMaker/t/META_for_testing.yml				test data for MakeMaker
 cpan/ExtUtils-MakeMaker/t/META_for_testing_tricky_version.yml				test data for MakeMaker
 cpan/ExtUtils-MakeMaker/t/metafile_data.t			See if META.yml handling works
 cpan/ExtUtils-MakeMaker/t/metafile_file.t			See if META.yml handling works
-cpan/ExtUtils-MakeMaker/t/min_perl_version.t			Test the MIN_PERL_VERSION argument to WriteMakefile.
+cpan/ExtUtils-MakeMaker/t/min_perl_version.t			Test the MIN_PERL_CORE_MINOR argument to WriteMakefile.
```

Comment:
pretty sure this is wrong.  MIN_PERL_VERSION accepts a fill version number and doesn't require a change, and no changes appear to have been made to `cpan/ExtUtils-MakeMaker/*`

Reply:
you are right this need to be restored to the previous definition

Reply:
fixed by 65ccb1427af285b0d15560552505b8ef5383802e